package.json

```diff
@@ -32,6 +32,7 @@
     "@ember/optional-features": "^0.6.3",
     "babel-eslint": "^10.0.1",
     "broccoli-asset-rev": "^2.7.0",
+    "ember-auto-import": "^1.5.2",
```

Comment:
Added `ember-auto-import` to easily import Node.js modules. In this case, it was to import `marked` for Markdown parsing.